### PR TITLE
docs: adds curl example for rpc (event, mempool, node) docs

### PR DESCRIPTION
**Pull request by holahula**
See title.

A follow-up PR will add cURL examples to the rest of the RPC docs

Part of ticket 699

Preview: https://website-git-holahula-docscurl-rpcs-ironfish.vercel.app/

### CODE REVIEW

File: content/documentation/rpc/mempool/get_transactions.mdx [hunk 1]
```diff
@@ -5,29 +5,70 @@ description: RPC Mempool GetTransactions | Iron Fish Documentation
...
+# Response
+{
+  "data": {
+    "serializedTransaction": "0102000...",
+    "position": 0,
+    "expiresIn": 14
+  }
+}
+{
+  "status": 200
```
Comment (dguenther) [inline content/documentation/rpc/mempool/get_transactions.mdx:71]: I think this is two separate objects now where it should be one

File: content/documentation/rpc/node/get_status.mdx [hunk 2]
```diff
@@ -18,13 +18,13 @@ Gets (and optionally streams) the node's status
...
+    status: 'started' | 'stopped' | 'error'     
     version: string
-    git: string
-    nodeName: string
+    git: string                                 
+    nodeName: string                            
   }
   cpu: {
-    cores: number
+    cores: number
```
Comment (dguenther) [inline content/documentation/rpc/node/get_status.mdx:27]: Some trailing whitespace here to delete

File: content/documentation/rpc/node/get_log_stream.mdx [hunk 3]
```diff
@@ -5,21 +5,39 @@ description: RPC Node | Iron Fish Documentation
...
+
+### Example
+
+```bash
+# Request
+curl -X POST http://localhost:8021/node/getLogStream
+
+# Response
+{
+  "data": {
```
Comment (dguenther) [inline content/documentation/rpc/node/get_log_stream.mdx:34]: Does this need a `status`?

File: content/documentation/rpc/event/on_gossip.mdx [hunk 4]
```diff
@@ -5,23 +5,43 @@ description: RPC Event | Iron Fish Documentation
...
+```
+
+### Example
+```bash
+# Request
+curl -X POST http://localhost:8021/event/onGossip
+
+# Response
+{
+  "data": {
```
Comment (dguenther) [inline content/documentation/rpc/event/on_gossip.mdx:36]: Does this need a `status`?

**Review by dguenther — APPROVED** [review]

**Review by NullSoldier — DISMISSED** [review]

**Comment by holahula** [comment]
Added `-N` to cURL options when streaming so output stream isn't buffered and it returns every chunk from the server when it comes in

File: content/documentation/rpc/node/get_log_stream.mdx (hunk 3 above)
Comment (holahula) [reply content/documentation/rpc/node/get_log_stream.mdx:34]: `status` isn't returned when using HTTP streaming

File: content/documentation/rpc/mempool/get_transactions.mdx (hunk 1 above)
Comment (holahula) [reply content/documentation/rpc/mempool/get_transactions.mdx:71]: it's returned as 2 separate objects from the HTTP adapter

**Pull request merged**